.github/workflows/update-node-version.yml

```diff
@@ -0,0 +1,58 @@
+name: Update Node version
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '15 12 * * *' # At 12:15 PM UTC every day
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  create-pr:
+    name: Create PR
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version-file: extensions/ql-vscode/.nvmrc
+          cache: 'npm'
+          cache-dependency-path: extensions/ql-vscode/package-lock.json
+      - name: Install dependencies
+        working-directory: extensions/ql-vscode
+        run: |
+          npm ci
+        shell: bash
```

Comment:
As far as I know, specifying `shell: bash` is unnecessary. But it doesn't hurt to be explicit and I have no problem with it.

It's unnecessary because `bash` is already [the default shell for `ubuntu-latest`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell). There's an edge case where if `bash` isn't available then it'll use `sh`, but it will always be available on hosted runnners. Also as far as I know, everything in this workflow behaves the same on `bash` and `sh`.

Reply:
I mostly copied this from our other workflows where we're also specifying `shell: bash`. I'm not sure why, but it seems like we should try to be consistent in either using it or not using it.

Reply:
Thanks for explaining. Sounds good to me.